### ISSUE TITLE
What is expected GPU speed-up?

### ISSUE THREAD

**Issue by chrisroat** [MouseLand/cellpose#89]
With a 40x128x128 single-channel image, I'm seeing a ~1.5x speed-up from using the GPU.    For a 201x256x256 single-channel image, I see about a ~3-4x speed-up.

Many workloads see much larger speed-ups on GPU.  What is normal for cellpose workloads?  See below for the smaller image.

```
model = models.Cellpose(gpu=True, model_type='nuclei')
results_nuclei = model.eval(data[80:120,:128,:128], channels=[0,0], diameter=None, do_3D=True)
```

NVIDIA Titan RTX
CUDA 10.1

python 3.7.6
cellpose 0.0.2.8
mxnet-cu101mkl 1.6.0

With CPU:
~1.4 iterations/sec
flows computed 213.80s
dynamics computed 214.85s
masks computed 215.18s

With GPU:
~2 iterations/sec
flows computed 149.93s
dynamics computed 151.14s
masks computed 151.48s

**Comment by marius10p** [comment]
Do you have cuDNN installed? I am surprised your bottleneck is not in dynamics or in masks, but maybe @carsen-stringer has accelerated that part sufficiently.

**Comment by carsen-stringer** [comment]
this is related to the augmentations and a single slice being run at a time (not using all the GPU memory). this should be faster now.
